workshop/exercise-6/README.md

```diff
@@ -1,86 +1,110 @@
-## (OPTIONAL, FOR ADVANCED USERS) Set up a Private Container Image Registry and Obtain Credentials
```

Comment:
I'm not sure why we need this exercise at all since it doesn't introduce any new features.

Reply:
This shows how to use knative/tekton to build & deploy -- all the previous labs just pulled an image already on dockerhub.

This is an interesting advanced exercise for longer labs or people who are interested in getting this set up with a private registry like IBM CR. It's marked as optional to indicate that it's not required.

An alternative would be to create a separate tekton lab that includes this information, similar to how we have a separate evening lab.